scripts/ci/check-connect-data.ts

```diff
@@ -7,25 +7,22 @@ import { comment, commit } from './helpers';
 const { exec } = require('./helpers');
 
 const AUTHENTICITY_BASE_URL = 'https://data.trezor.io';
-const authenticityPaths = {
-    T2B1: {
-        authenticity: 'firmware/t2b1/authenticity.json',
-        authenticityDev: 'firmware/t2b1/authenticity-dev.json',
-    },
-    T3B1: {
-        authenticity: 'firmware/t3b1/authenticity.json',
-        authenticityDev: 'firmware/t3b1/authenticity-dev.json',
-    },
-    T3T1: {
-        authenticity: 'firmware/t3t1/authenticity.json',
-        authenticityDev: 'firmware/t3t1/authenticity-dev.json',
-    },
-};
-
 const ROOT = path.join(__dirname, '..', '..');
 const CONFIG_FILE_PATH = path.join(ROOT, 'packages/connect/src/data/deviceAuthenticityConfig.ts');
+const DEVICES_SUPPORTING_DEVICE_AUTHENTICITY_CHECK = ['T2B1', 'T3B1', 'T3T1', 'T3W1'] as const;
+
+type DeviceModel = (typeof DEVICES_SUPPORTING_DEVICE_AUTHENTICITY_CHECK)[number];
```

Comment:
Given how few of them are, I wouldn't be afraid to duplicate value with type declaration, but no strong opinion, feel free to dismiss this :shrug: 
```suggestion
type DeviceModel = 'T2B1' | 'T3B1' | 'T3T1' | 'T3W1'
const DEVICES_SUPPORTING_DEVICE_AUTHENTICITY_CHECK: DeviceModel[] = ['T2B1', 'T3B1', 'T3T1', 'T3W1'];
```